src/Data/Functor/Linear.hs

```diff
@@ -13,8 +13,27 @@
 -- * Linear data functors should be thought of as containers of data.
 -- * Linear data applicative functors should be thought of as containers
 -- that can be zipped.
--- * Linear data traversible functors should be thought of as
+-- * Linear data traversable functors should be thought of as
 -- containers which store a finite number of values.
+--
+-- This module also defines 'genericTraverse' for types implementing
+-- 'Generics.Linear.Generic1':
+--
+-- >>> :set -XLinearTypes
+-- >>> :set -XTypeApplications
+-- >>> :set -XNoImplicitPrelude
+-- >>> :set -XDeriveAnyClass
+-- >>> import Prelude.Linear
+-- >>> import Data.Functor.Linear
+-- >>> import Generics.Linear
+-- >>> :{
+--  data Pair a = MkPair a a
+--    deriving (Show, Generic1)
```

Comment:
I'm putting a note here to not forget to update this with the proper code.

I suggest writing an example in `examples/` directory.